cartography/intel/aws/ecr.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+from cartography.util import run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
+
+def get_ecr_repositories(boto3_session, region):
```

Comment:
would it be possible to add the `@aws_handle_regions` decorator?
Sample: https://github.com/lyft/cartography/blob/master/cartography/intel/aws/eks.py

Reply:
Oh yeah, good idea.